media-gfx/openscad/openscad-2021.01.ebuild

```diff
@@ -69,9 +69,9 @@ src_prepare() {
 
 src_configure() {
 	if has ccache ${FEATURES}; then
-		eqmake5 "PREFIX = ${EROOT}/usr" "CONFIG += ccache" "${PN}.pro"
+		eqmake5 "PREFIX = ${ESYSROOT}/usr" "CONFIG += ccache" "${PN}.pro"
```

Comment:
ESYSROOT is definitely less wrong because it's valid and EROOT isn't, but if this is for the destination system, it's probably meant to be EPREFIX instead

ESYSROOT is where we can [find](https://dev.gentoo.org/~ulm/pms/head/pms.html#x1-720008.1) DEPEND-category dependencies installed during the build